Composer/packages/client/src/CreationFlow/DefineConversation/index.js

```diff
@@ -109,7 +136,14 @@ export function DefineConversation(props) {
             />
           </StackItem>
         </Stack>
-        {enableLocationBrowse && <LocationSelectContent onChange={updateLocation} allowOpeningBot={false} />}
+        {enableLocationBrowse && (
+          <LocationSelectContent
+            updateCurrentPath={updateCurrentPath}
```

Comment:
Instead of passing a function to child component to use. Consider passing a function that the child component can "fire event to".

Although the code is almost the same, probably just name changes, the philosophy of sending events make your code less prone to bugs, compare to passing a function. This is because less coupling between two components.